src/components/CForm/CForm.js

```diff
@@ -8,8 +8,10 @@ import {
 import { v4 } from 'uuid';
 import Element from '../Element';
 
+const textTypes = ['email', 'money', 'number', 'percentage', 'phone', 'url', 'password'];
```

Comment:
Do we need this anymore? From what I can see, `type` has `text` value for each subtype, and `context` is object with `type: email`.

Reply:
Yup, code leftover from previous solution. Nice catch :)